arraycontext/context.py

```diff
@@ -158,6 +158,13 @@ def _get_fake_numpy_namespace(self):
         from .fake_numpy import BaseFakeNumpyNamespace
         return BaseFakeNumpyNamespace(self)
 
+    def get_array_types(self):
+        """
+        Returns a :class:`tuple` of types that are valid base array classes
+        the context can operate on.
+        """
+        return ()
```

Comment:
Done.  9255f22

> Would it be sensible to only allow one type here, to avoid the splat above?

There is some value in keeping it tuple, default value is much nicer since the only intent is to perform type checking using it.